### PR TITLE
[Standalone for Native Component] ActivityIndicator 

### PR DESCRIPTION
**Pull request by VisibleMarkov**
PR Related to: #22990 

Changelog:
[Android][Changed] - All the imports connected to requireNativeComponent in ActivityIndicator was moved to  a seperate file.

Test Plan:
* [x]  yarn flow
* [x]  yarn lint
* [x] yarn prettier

### CODE REVIEW

File: Libraries/Components/ActivityIndicator/ActivityIndicatorNativeComponent.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const React = require('React');
```
[the remark this answers is not in the document]
Comment (VisibleMarkov) [reply Libraries/Components/ActivityIndicator/ActivityIndicatorNativeComponent.js]: Resolved in 8c0f044

File: Libraries/Components/ActivityIndicator/ActivityIndicatorNativeComponent.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const requireNativeComponent = require('requireNativeComponent');
+
+import type {ViewProps} from 'ViewPropTypes';
+import type {ViewStyleProp} from 'StyleSheet';
+import type {SyntheticEvent} from 'CoreEventTypes';
```
[the remark this answers is not in the document]
Comment (VisibleMarkov) [reply Libraries/Components/ActivityIndicator/ActivityIndicatorNativeComponent.js]: Resolved in 8c0f044

**Comment by VisibleMarkov** [comment]
@TheSavior, were the tests failing already or did this PR trigger them?
Also if there are any changes needed, please let me know and I'll change it ASAP (^_^)

**Comment by cpojer** [comment]
I'll ship this. Thanks for making this PR! :)

**Pull request closed without merging**